core/modules/editor/factory.js

```diff
@@ -206,13 +206,14 @@ function editTextWidgetFactory(toolbarEngine,nonToolbarEngine) {
 	Selectively refreshes the widget if needed. Returns true if the widget or any of its children needed re-rendering
 	*/
 	EditTextWidget.prototype.refresh = function(changedTiddlers) {
-		var changedAttributes = this.computeAttributes();
+		var changedAttributes = this.computeAttributes(),
+			editInfo = this.getEditInfo();
+		$tw.hooks.invokeHook("th-refreshing-input",this,editInfo,changedTiddlers,changedAttributes);
```

Comment:
This hook is specific to the edit text widget so it should be called something like `th-edit-text-widget-refreshing`

Reply:
Hi @Jermolene , I'll change it

Reply:
@Jermolene, the name is changed to `th-edit-text-widget-refreshing`

note: the edit widget is also accessible through this, it's a direct parent so no need for a second hook if one wants to access it